base62.py

```diff
@@ -40,6 +72,25 @@ def decode(b):
 
     return v
 
+
+def decodebytes(s):
+    """Decodes a bytestring of base62 data into a bytes object.
+
+    :param s: A bytestring to be decoded in base62
```

Comment:
Should accept a `str` (i.e. bytes in py2, text in py3) rather than `bytestring`.